shared/js/bluetooth_helper.js

```diff
@@ -175,11 +175,6 @@ var BluetoothHelper = function() {
     },
 
     getConnectedDevicesByProfile: function(profileID, cb) {
```

Comment:
though Callscreen is the only usecase, the similar convention of `getConnectedDevicesByProfile` abstraction is used in other bluetooth code such as 
https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/js/modules/bluetooth/bluetooth_connection_manager.js#L827

@evelynhung do you think it worth to change the naming here?